### ISSUE TITLE
Trying command from GUI is giving CM error: Multiple variation tags selected for the variation group "mode": {'find-performance', 'all-modes'} !

### ISSUE THREAD

**Issue by silhouette001** [mlcommons/ck#673]
![Screenshot from 2023-02-21 17-16-47](https://user-images.githubusercontent.com/84288594/220339071-3740e995-82df-4950-beb9-8c3b50305742.png)

**Comment by arjunsuresh** [comment]
Thank you @silhouette001 for raising this issue.

@gfursin We need to handle a special case in GUI. Sometimes a conflicting group variation can be called as a base of another variation. Here what is happening is that "all-modes" and "find-performance" are of the same group and "all-modes" is a base of "submission" variations. 

For now, I have changed the variation groups which was needed when I added "readme-populate" variation. So, now we have 3 submission modes.
1. "find-performance" - does a short performance run and populates the system config
2. "submission" - does a real performance and accuracy run
3. "readme-populate" - reuses the results from the previous submission run and adds `run commands` to the README files.
